backend/src/api_parsers/scopus_parser.py

```diff
@@ -1,14 +1,16 @@
-from src.api_parsers.exceptions import NoAuthorsException
+from src.api_parsers.exceptions import NoAuthorsException, MaxAuthorsReachedException
 from src.api_handlers.scopus.handler import ScopusHandler
 from src.api_parsers.models import Source, Publication, Author
 from requests.exceptions import HTTPError
-import json
 
 
 class ScopusParser:
-    def __init__(self, keywords: str):
+    def __init__(self, keywords: str, min_year: int = 2010, max_authors: int = 10):
         self.keywords = keywords
         self.handler = ScopusHandler()
+        self.authors = []
+        self.min_year = min_year
+        self.max_authors = max_authors
```

Comment:
Why doesnt this class have any public method? How would external service use this class?